doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -1,8 +1,15 @@
 
 
-# Decision Trees for Vulnerability Management
+# Decision by Whom about What 
 
-Viable decision guidance for vulnerability management should, at a minimum, consider the stakeholder groups, their potential decision outcomes, and the data needed for relevant decision points. The following sections address each of these parts, in turn, and should be taken as instructive examples. While we strive to make the examples realistic, we invite the community to engage and conduct empirical assessments to test examples. The following construction should be treated as an informed hypothesis rather than a conclusion.
+This section will define who we are trying to give decision advice to, and how we are scoping our advice on vulnerability management decisions. 
+Viable decision guidance for vulnerability management should, at a minimum, consider the stakeholder groups, their potential decision outcomes, and the data needed for relevant decision points. 
+This section covers the first of these three, and the following sections address the other parts in turn.
+The "who" is primarily about categories of stakeholders -- suppliers, deployers, and coordinators -- as well as their individual risk tolerances.
+The "what" is about the scope, both in how the affected system is defined and how much of the world an analyst should consider when estimating effects of a vulnerability. 
```

Comment:
Depending on there you are in the supply chain, you may be considered a supplier, deployer, and a coordinator at different stages of the vulnerability response.

Reply:
Are you suggesting we add this as a sentence in the ##Enumerating Stakeholders section intro?

Reply:
yes.  after this sentence: 
The "who" is primarily about categories of stakeholders -- suppliers, deployers, and coordinators -- as well as their individual risk tolerances. Depending on where you are in the supply chain, you may be considered a supplier, deployer, and a coordinator at different stages of a vulnerability response.

How's that look? ^^^

Reply:
I thought this needed a bit more in the way of example, so I added a few sentences, including this, to the first paragraph of ## Enumerating Stakeholders.

Thoughts?